CONTRIBUTING.md

```diff
@@ -6,3 +6,41 @@ We love your input! We want to make contributing to Roboflow Notebooks as easy a
 - When adding or editing notebooks, make sure to specify the versions of pip packages 📦 you are using.
 
 Please keep in mind that repository is still under construction 🚧. We will add more contribution guidelines soon.
+
+## Submitting a Bug Report 🐛
+
+If you spot a problem with Roboflow Notebooks, please submit a Bug Report!
+
+For us to start investigating a possible problem we need to be able to reproduce it ourselves first. We've created a few
+short guidelines below to help you provide what we need in order to get started.
+
+When asking a question, people will be better able to provide help if you provide **code** that they can easily
+understand and use to **reproduce** the problem. This is referred to by community members as creating
+a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces
+the problem should be:
+
+- ✅ **Minimal** – Use as little code as possible that still produces the same problem
+- ✅ **Complete** – Provide **all** parts someone else needs to reproduce your problem in the question itself
+- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem
+
+In addition to the above requirements, for the [Roboflow](https://roboflow.com/) team to provide assistance your code
+should be:
+
+- ✅ **Current** – Verify that your code is up-to-date with current
+  GitHub [main](https://github.com/roboflow/notebooks/tree/main), and if necessary `git pull` or `git clone` a new
+  copy to ensure your problem has not already been resolved by previous commits.
+- ✅ **Unmodified** – Your problem must be reproducible without any modifications to the codebase in this
+  repository. [Roboflow](https://roboflow.com/) does not provide support for custom code ⚠️. If custom code has been added, please post in the Community Forum for more assistance: https://discuss.roboflow.com
+
+If you believe your problem meets all of the above criteria, please close this issue and raise a new one using the 🐛
+**Bug Report** [template](https://github.com/roboflow/notebooks/issues/new/choose) and providing
+a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better
+understand and diagnose your problem.
+
+### How to File a Bug Report (VIDEO)
+[![Roboflow Support: Filing Bug Reports](https://img.youtube.com/vi/KE_SsEPaR_4/0.jpg)](https://www.youtube.com/watch?v=KE_SsEPaR_4)
```

Comment:
Video is a great addition, but we need to make the video thumbnail more eye-catching.